### PR TITLE
Add a trait that allows to use the new Play 2.6 Actions without much

### PR DESCRIPTION
**Pull request by raphaelbauer**
hassle.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,6 +72,20 @@ val ws = MockWS { ... }
...
+class MySpec extends FreeSpec with Matchers with MockWSHelpers {
+  ...
+}
+```
+
+or
+```scala
+import mockws.MockWSHelpers._
+```
+
```
Comment (yanns) [inline README.md]: 👍 for updating the doc!

**Comment by raphaelbauer** [comment]
Hey! My PR increased test coverage by 0.5% and codacy is still not happy ;)

I do not plan to add any more helpers... That's all we need in our projects... Why do you ask? Because of the naming?

**Comment by yanns** [comment]
You said it was a sketch.
But it's actually ready to merge, isn't it?

**Comment by raphaelbauer** [comment]
Yes. It's up to you :) Let me know if anything is missing. Or if anything should be improved!

**Comment by raphaelbauer** [comment]
Thanks Yann!

**Comment by yanns** [comment]
Thx for the PR!

**Comment by francisdb** [comment]
Would this `sys.addShutdownHook` not cause problems when continually running tests in an unforked setting? Everything will run in the same jvm that runs for a long time. Causing more an more actorsystems to be created but not cleaned up?

**Comment by francisdb** [comment]
Previously this was an object instead of a trait where this is less of a problem as there is only one instance.

**Comment by yanns** [comment]
@francisdb yep, there should be a general "shutdown" method, that could be used either by `afterAll()` test methods, or a shutdown hook on the Object.
I'll change that.

**Comment by yanns** [comment]
@francisdb can you please review https://github.com/leanovate/play-mockws/pull/31 ? Thx!

**Pull request merged**